README.md

```diff
@@ -76,6 +76,33 @@ values by `__SENSITIVE_DATA__` string. This feature is enabled by default but
 you can skip this (not recommanded) by setting the environment variable
 `LOGGER_USE_SENSITIVE_DATA_STREAM` to `false`.
 
+You also can add custom filter :
+```
+const sensitiveDataFragment = '(pass|password)'; // Will obfuscate 'pass' and 'password' data
+
+const newLogger = init({
+  logger: {
+    sensitiveDataFragment,
+  },
+});
+```
+
+Moreover, you can also add customize the way it replaces data :
+```
+const sensitiveDataPattern = [
+  {
+    regex: YOUR_NEW_REGEX,
+    substitute: SUBSTITUTION_CONTENT,
```

Comment:
Do we really need to specify a substitute? the default one should be enough? Don't you think?

Reply:
Substitute is indeed replacement pattern like `"$1":"__SENSITIVE_DATA__"`, and it is not the same depending on cases, sometimes we need quotes, for other ones we don't necessarily.

Reply:
If it is just to avoid double quote in the equal case, I keep thinking it is not necessary.